### PR TITLE
DiscordBridge v2.0

### PR DESCRIPTION
**Pull request by DiamondIceNS**

### CODE REVIEW

**Comment by jacobwgillespie** [comment]
Looks awesome 🎉 

Merge away!

Which reminds me, another housekeeping thing: if you open a PR and want to wait for a review, once the reviewer has approved the PR, they will let you merge it - that way, stuff doesn't get accidentally merged too early in the case where you still want to make changes to the branch.

tl;dr - if you open a PR, you will typically be the one to merge it

**Review by jacobwgillespie — APPROVED** [review]

**Comment by DiamondIceNS** [comment]
Hooray, learning!

Thanks for being patient with me, I'll keep all of this in mind going forward. These are the taken for granted details that my school never bothered to teach me and just expected me to figure out.

**Pull request merged**
